examples/Cpp/ReflexGame/src/ReflexGame.lf

```diff
@@ -114,13 +107,16 @@ reactor GetUserInput {
       total_time += time_in_ms;
       // Reset the prompt_time to indicate that there is no new prompt.
       prompt_time = reactor::TimePoint::min();
-      // Trigger another prompt.
-      another.set();
+      // Request another prompt.
+      request_prompt.set();
     }
   =}
 
+  reaction(quit) {=
+    environment()->sync_shutdown();
```

Comment:
What does this do?

Reply:
This requests a shutdown at the next microstep.